recipe/meta.yaml

```diff
@@ -45,11 +45,10 @@ test:
     - if not exist %LIBRARY_BIN%\gtest.lib exit 1           # [win]
     - if not exist %LIBRARY_BIN%\gtest_main.lib exit 1      # [win]
     - if not exist %LIBRARY_BIN%\gtest_dll.dll exit 1       # [win]
-    - if not exist %LIBRARY_BIN%\gtest_md.lib exit 1        # [win]
-    - if not exist %LIBRARY_BIN%\gtest_md.lib exit 1        # [win]
```

Comment:
I'm guessing one of these was suppose to be `.dll`. Can we add such a test below for the new filename?

Reply:
Great catch!